src/modules/awake/Awake/Core/Models/SystemPowerState.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Awake.Core.Models
+{
+    // Maps to the OS power state.
+    // See documentation: https://docs.microsoft.com/windows/win32/power/system-power-states
+    public enum SystemPowerState
+    {
+        PowerSystemUnspecified = 0,
+        PowerSystemWorking = 1,
+        PowerSystemSleeping1 = 2,
+        PowerSystemSleeping2 = 3,
+        PowerSystemSleeping3 = 4,
```

Comment:
Just wondering, do you need to differentiate them? I'm not fully aware of the details, but the docs you mentioned have S1-S3 combined, and it looks like there isn't much difference between them.